IPython/utils/tests/test_module_paths.py

```diff
@@ -114,7 +125,15 @@ def test_find_module_1():
 
 def test_find_module_2():
     """Testing sys.path that is empty"""
-    nt.assert_is_none(mp.find_module("xmod", []))
+    # on 3.5 we cannot pass PATH top importlib, 
```

Comment:
top -> to, and don't capitalise PATH, it makes it look like an environment variable